cime_config/acme/allactive/config_pes.xml.cime2

```diff
@@ -469,6 +474,7 @@
 <NTASKS_GLC>2048</NTASKS_GLC> <NTHRDS_GLC>16</NTHRDS_GLC> <ROOTPE_GLC>0</ROOTPE_GLC> <NINST_GLC>1</NINST_GLC>
 <NTASKS_WAV>2048</NTASKS_WAV> <NTHRDS_WAV>16</NTHRDS_WAV> <ROOTPE_WAV>0</ROOTPE_WAV> <NINST_WAV>1</NINST_WAV>
 <NTASKS_CPL>2048</NTASKS_CPL> <NTHRDS_CPL>16</NTHRDS_CPL> <ROOTPE_CPL>0</ROOTPE_CPL>
+<PIO_NUMTASKS>-1</PIO_NUMTASKS><PIO_STRIDE>64</PIO_STRIDE> <PIO_TYPENAME>pnetcdf</PIO_TYPENAME> <PIO_ROOT>1</PIO_ROOT>
```

Comment:
Ignore any file ending in "cime2".  I kept those around in case I needed to look at them.